error_reporting/main_test.py

```diff
@@ -17,7 +17,9 @@
 import main
 
 
-@mock.patch("fluent.event")
-def test_error_sends(event_mock):
+@mock.patch("main.error_reporting")
+def test_error_sends(error_reporting_mock):
+    client_mock = mock.Mock()
```

Comment:
It doesn't test nothing, it tests the code path, which it doesn't without the mock. `wraps` seems like it cleans up the mocking code. However, can see both sides and not worth arguing about so made change.